packages/react-storage/src/components/StorageBrowser/composables/DataTable/DataTable.tsx

```diff
@@ -18,11 +18,13 @@ export interface DataTableRow {
 
 export interface DataTableProps {
   headers: WithKey<DataTableHeader>[];
+  isLoading?: boolean;
   rows: WithKey<DataTableRow>[];
```

Comment:
If all we are doing with isLoading is returning an empty array, we should remove `isLoading` and make `rows` optional (note, most React `props` should be optional) and only provide a value for `rows` once the loading is complete